charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -66,6 +69,8 @@ spec:
               name: config-volume
             - mountPath: {{ .Values.server.config.path }}/catalog
               name: catalog-volume
+            - mountPath: /etc/trino/schemas
```

Comment:
Let's make the path configurable. Otherwise the only way people who want this feature would be able to use it is by reading the chart and seeing what path the files get mounted to.